Server/src/main/java/com/assignment/concurrent/domain/Point.java

```diff
@@ -3,39 +3,59 @@
 import lombok.Data;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import java.util.concurrent.locks.ReentrantLock;
 
 @Data
 public class Point {
```

Comment:
Ka chun, this class is created by clement ady, conflict with yours, what is your purpose of this class, testing only?

Reply:
if i not mistake he pulled clement branch

Reply:
this branch was code on top of Clement's point branch. 
If merge conflict happens at Point.class , please prioritize the point branch.